### DISCUSSION TITLE
How to login with a onedrive 365 account provided by my university?

### DISCUSSION THREAD

**Discussion by ykzhang0126** [abraunegg/onedrive#2564] [Q&A] [answered]
I have a onedrive 365 account provided by my university (https://itsc.nju.edu.cn/4a/8e/c45143a543374/pagem.htm). On Windows, I can login with it by using the email "xxx@365.nju.edu.cn", and it will redirect to the university account verification page. However, when I am trying to login on ubuntu using command line, the output login url prompt that "This username may be incorrect. Make sure you typed it correctly. Otherwise, contact your admin.". What is the correct way to login?
<img width="659" alt="Capture2" src="https://github.com/abraunegg/onedrive/assets/32958459/5c8d01e4-0048-4aa2-9c7a-643953406c4f">
<img width="427" alt="Capture1" src="https://github.com/abraunegg/onedrive/assets/32958459/2590771a-acbb-4b8e-aa04-23ab8967808a">
<img width="238" alt="Capture" src="https://github.com/abraunegg/onedrive/assets/32958459/fa60333f-4480-4700-ab49-6cea9874ad6d">

**Comment by abraunegg** [comment, marked as answer]
@ykzhang0126 
Firstly, as you are using Ubuntu - what application version are you running? Please can you provide the output of `onedrive --version`


Secondly, you will need to speak with your University. You *may* need to configure an 'azure_tenant_id' in your configuration file to ensure the client is authenticating against your University system, and not the Global Azure Active Directory Platform.

![image](https://github.com/abraunegg/onedrive/assets/4956234/3417a7fc-e06d-4c0e-a501-1dc57a9b4b71)
